src/site/content/en/blog/aria-poison-or-antidote/index.md

```diff
@@ -0,0 +1,283 @@
+    the appropriate action (or even open another menu).
++   If the user presses a key that should do something else, don't eat that!
+    Throw it back to the page as nature intended. For example, our menubar
+    doesn't need the <kbd>Tab</kbd> key, so throw it back! This is hard to get
+    right, and authors often mess it up. For example, the menubar needs arrow
+    keys, but not <kbd>Alt</kbd>+<kbd>Arrow</kbd> or
+    <kbd>Command</kbd>+</kbd>Arrow</kbd>. Those are shortcuts for moving to the
+    previous/next page in the web history of your browser tab. If the author
+    isn't careful, the menubar will eat those. This kind of bug happens a lot,
+    and we haven't even started with ARIA yet!
+
+### Screen reader access to our menubar
+
+Our menu bar was created with duct tape and divs. As a result, a screen reader has no idea what any
+of it is. The background color for the active item is just a color. The menu item divs are just
+plain objects with no particular meaning. As such, a user of our menubar doesn't get any
+instructions of what keys to press or what item they're on.
+
+But that's no fair! The menu bar acts just fine for the sighted user. 
+
+ARIA to the rescue. ARIA lets us pretend to the screen reader and convince it that focus is in a
+menu bar. If the author does everything right, our custom menu bar will look to the screen reader
+just like a menu bar in a native desktop application.
+
+Our first, ahem, ARIA lie, is to use the `aria-activedescendant` attribute, and set it to the ID of
+the currently active menuitem, being careful to update it whenever it changes. For example,
+`aria-activedescendant="settings-menuitem"`. This little white lie causes the screen reader to
+consider our ARIA active item as the focus, which is read aloud or shown on a Braille display. 
+
+A word on the _ancestor_, _parent_, and _descendant_ terminology. The term descendant refers to the
+fact that an item is contained somewhere inside of another. The reverse term is ancestor, which is
+to say an item is contained by ancestors. For the next container up/down, these may use the more
+specific terms parent/child. For example, imagine a document with a paragraph that has a link
+inside. The link's parent is a paragraph, but it also has the document as an ancestor. And in
+reverse, the document may have many paragraph children, each with links. The links are all
+descendants of the grandparent document.
+
+Back to `aria-activedescendant`. By using it to point from the focused menubar to a specific menu
+item,  the screen reader now knows where the user has moved, but nothing else about the object. What
+is this div thing anyway? That's where the role attribute comes in. We use `role="menubar"` on the
+containing element for the entire thing, then we use `role="menu"` on groups of items, and
+`role="menuitem"` on … drumroll … the individual menu items.
```

Comment:
```suggestion
`role="menuitem"` on (drumroll) the individual menu items.
```

Reply:
The mac screen reader rolls through the periods of elipsis as though they're not present. A pause would be nice. How does it handle parens?

Reply:
```suggestion
`role="menuitem"` on—drumroll—the individual menu items.
```

Reply:
When I compare `on drumroll the` with `on… drumroll… the` I hear a pause